integration/datasync-mongodb-model.graphql

```diff
@@ -1,6 +1,8 @@
 """
 @model
-@datasync
+@datasync(
+  ttl: 5184000
```

Comment:
Does it mean TTL for soft deletes or TTL for the conflict. 
Would they be handled by the same value now? 

Any explanation what TTL means here will be good.

Reply:
This is the ttl for the base table, both for conflicts as well as soft deletes

Reply:
TTL for base table would be setup in schema and TTL for delta table would be given in createDataSyncAPI, I could not find a better way to put this because createDataSyncAPI has to do both i.e. just delta queries AND delta queries with conflict resolution, and we need base table TTL for both of these cases, Only alternative is to pass a map of model name to base table TTL separately to createDataSyncAPI

That maybe sounds like it might be problematic and as such I might need some time to fix it up

Reply:
This is good. If you think about it those values need to be connected